java/src/jmri/server/json/block/JsonBlockHttpService.java

```diff
@@ -63,7 +63,7 @@ public ObjectNode doGet(Block block, String name, String type, JsonRequest reque
         } else if (bv instanceof jmri.IdTag) {
             ObjectNode idTagValue = idTagService.doGet((jmri.IdTag) bv, name, IDTAG, request);
             data.set(VALUE, idTagValue);
-        } else if (bv instanceof Reportable) {
+        } else if (bv instanceof jmri.Reporter) {
```

Comment:
Please leave instanceof alone and change the cast instead.

Reply:
@rhwood has this right.  Reportable and reporter are not the same thing.  Reportable objects are returned by reporters.

Reply:
The method it's being passed to is expecting a Reporter, not a Reportable?

![8358](https://user-images.githubusercontent.com/39652002/79403563-308bc400-7f87-11ea-811f-7f4072c19ed5.png)